packages/shared/src/components/auth/OnboardingHeadline.tsx

```diff
@@ -1,30 +1,57 @@
 import React, { ReactElement } from 'react';
 import classNames from 'classnames';
-import { OnboardingTitleGradient } from '../onboarding/common';
+import { OnboardingGradientClasses } from '../onboarding/common';
 
 interface ClassName {
+  pretitle?: string;
   title?: string;
   description?: string;
 }
 
 interface OnboardingHeadlineProps {
+  pretitle?: string;
+  title?: string;
+  description?: string;
   className?: ClassName;
 }
 
-export function OnboardingHeadline({
-  className = {},
-}: OnboardingHeadlineProps): ReactElement {
-  return (
-    <>
-      <OnboardingTitleGradient className={classNames('mb-4', className?.title)}>
-        Where developers suffer together
-      </OnboardingTitleGradient>
-
-      <h2 className={className?.description}>
-        We know how hard it is to be a developer. It doesn&apos;t have to be.
+const defaultTitle = 'Where developers suffer together';
+const defaultDescription = `We know how hard it is to be a developer. It doesn't have to be.
         Personalized news feed, dev community and search, much better than
-        what&apos;s out there. Maybe ;)
-      </h2>
-    </>
+        what's out there. Maybe ;)`;
+
+export function OnboardingHeadline(
+  {
+    pretitle,
+    title = defaultTitle,
+    description = defaultDescription,
+    className,
+  }: OnboardingHeadlineProps = {
+    title: defaultTitle,
+    description: defaultDescription,
+  },
```

Comment:
We are setting the fallback value up above, would that not work alone?

```suggestion
  }: OnboardingHeadlineProps,
```

Reply:
It does, oversight when refactoring :)